discord/commands/permissions.py

```diff
@@ -135,6 +143,38 @@ def decorator(func: Callable):
 
     return decorator
 
+def cog_has_role(item: Union[int, str], guild_id: int = None):
+    """Modified version of has_role that applies to all functions in a cog.
+    
+    .. versionadded:: 2.0
+    
+    Parameters
+    -----------
+    parent

```

Comment:
What is the parent argument here?

Reply:
I think that's an old doc string. I have since removed the parent argument from every cog_* function in this PR.